msgcntr/messageforums-app/src/java/org/sakaiproject/tool/messageforums/ForumsEmailService.java

```diff
@@ -345,4 +348,30 @@ private boolean isValidEmail(String email) {
 		
 	}
 
+	private String getAnonAwareAuthor(Message message)
+	{
+		if (isUseAnonymousId(message.getTopic()))
```

Comment:
It comes from the object's constructor, but a null check wouldn't hurt. Though if it was null, we'd probably throw an IllegalArgumentException. IAE / NPE = nbd.